cfg/phab_tracker_only.json

```diff
@@ -0,0 +1,127 @@
+{
+  // verbose
+  "simulation": false,
+  "print_timing": false,
+  "use_canvas": true,
+  "use_debug_view": true,  // draw rejected & dropped features on canvas
+  "async_run": false, // turn this off in benchmarking
+
+  // visualization (tracker view) option
+  "draw_OOS": true,
+
+  // memory
+  "memory": {
+    // Values if mapper is not enabled
+    "max_features": 200,
+    "max_groups": 100
+    // Values if mapper is enabled
+    //"max_features": 20000,
+    //"max_groups": 3500
+  },
+
+
+  "camera_cfg": {
```

Comment:
The user shouldn't need to give details about the camera calibration if they just want to run the tracker. We can delete this block from the config file by making a few edits to `factory.cpp`:
- Check if `camera_cfg` is in the `Json::Value &cfg`.
- Only create the camera object if it's there.